packages/server-express/src/middlewares/fixtures/jwt-token.ts

```diff
@@ -0,0 +1,17 @@
+import { JWT, JWK, JWKS } from '@panva/jose';
+
+const keyRS256 = JWK.generateSync('RSA', 2048, { use: 'sig', alg: 'RS256' });
+
+export const jwksStore = new JWKS.KeyStore([keyRS256]);
+
+export const createToken = (options: { issuer: string; audience: string }) =>
```

Comment:
Nit: exporting in the end `exports { jwksStore, createToken }` helps (at least me) when scanning the code :)